lua-aws/engines/crypto/internal.lua

```diff
@@ -0,0 +1,12 @@
+local util = require('lua-aws.util')
+local sha2 = require('lua-aws.deps.sha2')
+
+return {
+    hash_sha256 = function (message, output_format)
```

Comment:
I don't think you need to set the anonymous function here. Does this work?

hash_sha256 = sha2.hash256